ajax/libs/react-mdl/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "react-mdl",
+  "filename": "ReactMDL.min.js",
+  "version": "1.10.3",
+  "description": "React Components for Material Design Lite",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/react-mdl/react-mdl"
+  },
+  "homepage": "https://react-mdl.github.io/react-mdl/",
+  "keywords": [
+    "react",
+    "react-component",
+    "mdl",
+    "material-design",
+    "badge",
+    "button",
+    "card",
+    "checkbox",
+    "data-table",
+    "fab-button",
+    "icon",
+    "icon-button",
+    "icon-toggle",
+    "radio",
+    "radio-group",
+    "switch",
+    "textfield",
+    "progress",
+    "progress-bar",
+    "spinner",
+    "slider",
+    "tooltip",
+    "menu",
+    "dropdown",
+    "Layout",
+    "Drawer",
+    "Grid",
+    "Tabs"
+  ],
+  "author": {
+    "name": "Tommy Leunen",
+    "email": "tommy.leunen@gmail.com",
+    "url": "https://tommyleunen.com/"
```

Comment:
https://www.tommyleunen.com/ is better because the origin link always redirect to this.
What do you think?
Thanks.